recipes/cf_xarray/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "cf_xarray" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xarray-contrib/cf-xarray/archive/v{{ version }}.tar.gz
```

May want to use archives from PyPI

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```